fiberi18n/i18n.go

```diff
@@ -63,42 +63,54 @@ func (c *Config) initLocalizerMap() {
 	if _, ok := localizerMap.Load(lang); !ok {
 		localizerMap.Store(lang, i18n.NewLocalizer(c.bundle, lang))
 	}
-	c.localizerMap = localizerMap
+	c.mu.Lock()
+	defer c.mu.Unlock()
+	newLocalizerMap := &sync.Map{}
```

Comment:
why do we need a copy of sync.map here and can't use the one already initialized in config?